### PR TITLE
Adding manifests for manager rest

### PR DESCRIPTION
**Pull request by johnugeorge**
Adding manifests for rest server for API manger

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/542)

### CODE REVIEW

File: scripts/v1alpha2/deploy.sh [hunk 1]
```diff
@@ -33,6 +33,8 @@ kubectl apply -f manifests/v1alpha2/katib-controller/metricsControllerConfigMap.
 kubectl apply -f manifests/v1alpha2/katib-controller/katib-controller.yaml
 kubectl apply -f manifests/v1alpha2/katib/manager/service.yaml
 kubectl apply -f manifests/v1alpha2/katib/manager/deployment.yaml
+kubectl apply -f manifests/v1alpha2/katib/manager-rest/service.yaml
+kubectl apply -f manifests/v1alpha2/katib/manager-rest/deployment.yaml
```
Comment (hougangliu) [inline scripts/v1alpha2/deploy.sh]: I think you need use new build image to replace pre-defined one in manifests/v1alpha2/katib/manager-rest/deployment.yaml in run-tests.sh to make test pass

**Comment by gaocegege** [comment]
/hold 

for @hougangliu 's advice.

**Pull request merged**